<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Try Catch</title>
    <script src="script.js" defer></script>
  </head>
  <body style="font-family: cursive">
    <h1>Try Catch</h1>
    <p>The try...catch statement in JavaScript is used to handle errors or exceptions gracefully without crashing the program. It allows you to "try" a block of code that might throw an error and "catch" the error to handle it.</p>
    <p><b># try block:</b> Contains the code that might throw an exception. <br>
     <b> # catch block: </b>Executes if an error occurs in the try block. The error is passed as a parameter to the catch block. <br>
      <b># finally block (optional):</b> Runs after the try and catch blocks, regardless of the outcome. Itâ€™s typically used for cleanup tasks.</p>
  </body>
</html>